scaife_viewer/settings.py

```diff
@@ -268,6 +268,7 @@
 CACHES = {
     "default": {
         "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
```

Comment:
`LocMemCache` concerns me with multiple gunicorn processes. If you've tested this out, then it should be good to go!

Reply:
@mjhea0 i think the worse case is the other process computes it as well, unlike an application with transactions the data shouldn't be different between processes.